app/build.gradle

```diff
@@ -3,13 +3,13 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 30
     buildToolsVersion "29.0.2"
 
     defaultConfig {
-        applicationId "com.banuba.sdk.example.quickstart_cpp"
+        applicationId "com.banuba.sdk.example.quickstart_c_api"
         minSdkVersion 24
-        targetSdkVersion 28
+        targetSdkVersion 30
```

Comment:
consider upgrading to API 31

Reply:
With version API 31 the example does not build